EdFi.Ods.Utilities.Migration/Scripts/02 Upgrade/v34_to_v50/01 Structure/1240-UpdateViewsForV50.sql

```diff
@@ -0,0 +1,65 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+
+IF EXISTS (SELECT * FROM sys.views WHERE object_id = OBJECT_ID(N'[auth].[EducationOrganizationIdentifiers]'))
+    DROP VIEW auth.EducationOrganizationIdentifiers;
+GO
+
+CREATE VIEW [auth].[EducationOrganizationIdentifiers]
```

Comment:
Note that if the implementation had previously customized this view (such as for a new Education Organization subclass, like TPDM does), this migration will effectively overwrite their modification.  Not sure if its worth documenting that @vimayya

Reply:
@blmeyers added a note about this on https://techdocs.ed-fi.org/display/EDFITOOLS/Migration+Utility (Step 8)